db/seeds.rb

```diff
@@ -12,8 +12,6 @@
 OrderStatus.create(name: "Complete")
 OrderStatus.create(name: "Reconciled")
 
-Affiliate.find_or_create_by(name: "Other")
```

Comment:
Do we want to get rid of the default "Other" in seeds? Or is it added somewhere else?